lib/grp.gd

```diff
@@ -2039,25 +2039,26 @@ DeclareAttribute( "ComputedAgemos", IsGroup, "mutable" );
 
 #############################################################################
 ##
-#A  RadicalGroup( <G> )
+#A  SolvableRadical( <G> )
 ##
-##  <#GAPDoc Label="RadicalGroup">
+##  <#GAPDoc Label="SolvableRadical">
 ##  <ManSection>
-##  <Attr Name="RadicalGroup" Arg='G'/>
+##  <Attr Name="SolvableRadical" Arg='G'/>
 ##
 ##  <Description>
-##  is the radical of <A>G</A>, i.e., the largest solvable normal subgroup of <A>G</A>.
+##  is the solvable radical of the group <A>G</A>,
```

Comment:
Do you think it would be a good idea to add an index entry `RadicalGroup` that takes the user directly to the `SolvableRadical` manual entry? I'm not sure, but thought I thought I'd suggest it.
```suggestion
##  <Index Key="RadicalGroup"><C>RadicalGroup</C></Index>
##  is the solvable radical of the group <A>G</A>,
```

Reply:
See the remark about `doc/ref/obsolete.xml`: Now there is an index entry that points to a section that lists name changes.